src/modules/service-catalog/ServiceCatalogItemPage.tsx

```diff
@@ -97,43 +32,65 @@ export function ServiceCatalogItemPage({
   brandId,
 }: ServiceCatalogItemPageProps) {
   const serviceCatalogItem = useServiceCatalogItem(serviceCatalogItemId);
-  const { requestFields, handleChange } = useItemFormFields(
+  const {
+    requestFields,
+    associatedLookupField,
+    setRequestFields,
+    handleChange,
+  } = useItemFormFields(serviceCatalogItem, baseLocale);
+  const { submitServiceItemRequest } = useServiceFormSubmit(
     serviceCatalogItem,
+    requestFields,
+    associatedLookupField,
     baseLocale
   );
-  const { t } = useTranslation();
+
+  const handleRequestSubmit = async (e: React.FormEvent<HTMLFormElement>) => {
+    e.preventDefault();
+
+    const response = await submitServiceItemRequest();
+    if (response && !response.ok) {
```

Comment:
This wasn't specified in the Jira story, but we can get different errors from the API, like a 422 when there are user errors or other errors like a 500 for other reasons, and we should handle them differently.

We should check the status code, if it is a 422 we should leverage the already implemented logic, otherwise, we should probably just show an error notification

Reply:
ok changed